src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -140,6 +140,22 @@ protected override Task CreateVersionsAsync(CancellationToken cancellationToken)
             var latestPrerelease = allVersionsAllowed.FirstOrDefault(v => v.version.IsPrerelease);
             var latestStableVersion = allVersionsAllowed.FirstOrDefault(v => !v.version.IsPrerelease);
 
+            if (_nugetProjects.Any()) {
+                if (_nugetProjects.FirstOrDefault().ProjectStyle.Equals(ProjectModel.ProjectStyle.PackageReference) && installedDependency != null)
+                {
+                    VersionRange installedVersionRange = VersionRange.Parse(installedDependency?.VersionRange?.OriginalString, true);
+                    NuGetVersion bestVersion = installedVersionRange.FindBestMatch(allVersionsAllowed.Select(v => v.version));
+                    DisplayVersion displayVersion = new DisplayVersion(installedVersionRange, bestVersion, additionalInfo: null);
+                    _versions.Add(displayVersion);
+                }
+                else if (installedDependency != null)
+                {
+                    VersionRange installedVersionRange = VersionRange.Parse(installedDependency?.VersionRange?.OriginalString, false);
```

Comment:
VersionRange uses the ?. operator. That means that there is a case where installedDependency can have a VersionRange which is null. VersionRange.Parse cannot handle parsing a null and it will throw a ArgumentNullException if you try to pass it a null.

Reply:
You're right, looking at the code this `VersionRange` will always have a value, im deleting the `?` operator